python/tvm/relay/testing/tf.py

```diff
@@ -184,10 +184,15 @@ def get_workload_official(model_url, model_sub_path):
     dir_path = os.path.dirname(model_path)
 
     import tarfile
+    import zipfile
```

Comment:
suggest to move this imports inside if or elif. let it import based on file type.